README.md

```diff
@@ -196,10 +207,9 @@ export type SinglePaymentRequest = {
 };
 ```
 
-After performing the transaction you can proceed by submitting it to our api.
-For that you can directly call our `approve-transaction` api endpoint with request body as follows.
+After performing the transaction you can proceed by submitting it to our API
 
-Approve transaction request body has the following signature:
+For that you can directly call our `approve-transaction` API endpoint with the request body as follows:
```

Comment:
this endpoint has changed, now it's "submit"

Reply:
Now updated.